gn3/api/correlation.py

```diff
@@ -22,11 +40,11 @@ def compute_sample_r(corr_method="pearson"):
 
     # xtodo move code below to compute_all_sampl correlation
     this_trait_data = correlation_input.get("this_trait")
-    target_datasets = correlation_input.get("target_dataset")
+    target_dataset_data = correlation_input.get("target_dataset")
```

Comment:
Something I usually do in Python when naming things I intend to remain hidden would be to start it with an underscore. I'd call `target_dataset_data` just `_dataset_data`. That's entirely a matter of style though.